Client/Frontend/Browser/AboutHomeHandler.swift

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+struct AboutHomeHandler {
```

Comment:
Can you add another class-level comment here briefly explaining why we have this?